pkg/apis/servicecatalog/validation/broker_test.go

```diff
@@ -519,13 +519,14 @@ func TestValidateClusterServiceBroker(t *testing.T) {
 	}
 
 	for _, tc := range cases {
-		errs := ValidateClusterServiceBroker(tc.broker)
-		if len(errs) != 0 && tc.valid {
-			t.Errorf("%v: unexpected error: %v", tc.name, errs)
-			return
-		} else if len(errs) == 0 && !tc.valid {
-			t.Errorf("%v: unexpected success", tc.name)
-		}
+		t.Run(tc.name, func(t *testing.T) {
+			errs := ValidateClusterServiceBroker(tc.broker)
+			if len(errs) != 0 && tc.valid {
+				t.Errorf("%v: unexpected error: %v", tc.name, errs)
```

Comment:
No need to print `tc.name` anymore?

Reply:
Oops, yeah, I didn't notice that. I've now removed it everywhere.